src/services/DemoApi.ts

```diff
@@ -124,6 +125,45 @@ export class DemoApi {
     await commands.executeCommand(COMMAND.runById, id);
   }
 
+  /**
+   * Handles the redirect request to open a VSCode URL.
+   *
+   * @param req - The HTTP request object.
+   * @param res - The HTTP response object.
+   */
+  private static async redirect(req: Request, res: Response) {
+    const path = req.query.path as string;
+
+    if (!path) {
+      res.status(400).send('Missing path parameter');
+      return;
+    }
+
+    // Validate that the path is a vscode:// URL
+    if (!path.startsWith('vscode://')) {
```

Comment:
You can remove the `vscode://` check, otherwise we cannot use insider and other editors